### DISCUSSION TITLE
Optimized Bayesian GPLVM to use as encoder for new data?

### DISCUSSION THREAD

**Discussion by norberttoth398** [cornellius-gp/gpytorch#2035] [Q&A]
Hi all

Just have a quick Q about GPLVM's with gpytorch I couldn't find an answer for - are they possible to be used as autoencoders for example to encode new data (not used for optimisation) to the latent space?

Thank you!

**Comment by gpleiss** [comment]
Sorry for the slow reply @norberttoth398 ! This is not currently possible in GPyTorch. In fact, I'm not aware of any papers that propose a method to dynamically encode new data to the latent space (but I'm not an expert on GPLVMs).
